default/stringtables/en.txt

```diff
@@ -8652,6 +8673,122 @@ Homeworld:
 A swamp world with a dense atmosphere and fairly dense biosphere.
 '''
 
+SP_THENIAN
+Thenian
+
+SP_THENIAN_GAMEPLAY_DESC
+'''Thoughtful gasbags.
+[[encyclopedia GASEOUS_SPECIES_TITLE]]
+'''
+
+SP_THENIAN_DESC
+'''
+At home in the boundary layers between the ultra high pressured inner atmosphere of their [[PT_GASGIANT]] and the outermost metallic helium core. They take their sustainance from the magnetic differential currents and are subject to periodical population dieoffs which is completely natural and of no concern to themselves whatsoever. Whenever the maximum population density is reached, another dieoff occurs so they can grow anew with unrelenting vigor. (Current [[metertype METER_POPULATION]] of [[SP_THENIAN]] colonies will be cut in half the turn after the maximum is reached.)
```

Comment:
neither is it my first. and i hate to use spell-checking in a programming environment, but yeah probably i should put text fragments like these through some sort of style checker next time – which is what i like to do anyway when translating stuff, but since this was original material .. ok i'll correct the typos.

![sustenance vs  sustainance](https://user-images.githubusercontent.com/37298400/152043092-b7f729c6-3073-4b8e-b25e-63dcb143c071.png)

english spelling is tough stuff indeed .. no wonder they have those spelling contests over there - seems i got it right in this case, but i can see it is similar like maintainance ..